### DISCUSSION TITLE
[idea] message speaker to revise their proposal

### DISCUSSION THREAD

**Discussion by ckworkshop** [conference-hall/conference-hall#285] [Ideas and features]
Would love to communicate with potential speakers and next steps.

**Comment by bpetetot** [comment]
Hey! Thanks for the suggestion—definitely a useful feature to have a communication channel with speakers. We’re actually thinking about it, but it’s not a simple one to implement. It’ll need some design work to get it right and ensure it integrates smoothly into the platform, so it might take some time. We’ll keep it in mind as we continue to develop. Thanks again for the idea!
